lib/phx_tools_web/live/phx_tools_live/linux.html.heex

```diff
@@ -0,0 +1,42 @@
+<div class="solved-height font-normal">
+  <div class="w-full h-full flex items-center justify-center">
+    <div class="block lg:mt-[3%] ">
+      <h1 class="text-white text-center lg:text-[24px] md:text-[22px] sm:text-[20px] lg:mb-[5%] md:mb-[2%] sm:mb-[2%]">
+        Linux installation process
+      </h1>
+      <div class="lg:w-[671px] lg:h-[272px] md:w-[661px] md:h-[290px] sm:w-[330px] sm:h-[350px] border-[4px] shadow-[#28177E] shadow-md border-[#211666] rounded-md lg:mx-[2%]">
+        <div class="text-start px-[3%] lg:text-[20px] md:text-[18px] sm:text-[15px]">
+          <h1 class="text-white lg:mt-5 md:mt-5 sm:mt-4">
+            1. Press the button to copy this command to your clipboard:
+          </h1>
+          <div class="flex items-center justify-evenly border-[1px] border-[#755FFF] py-1 mx-[4%] lg:mt-6 md:mt-5 sm:mt-4 rounded-lg">
+            <h1 id="tool-installation" class="text-white  text-[16px] font-thin">
+              /bin/bash -c "$(curl -fsSL https://phx-tools-test.fly.dev/linux.sh)"
+            </h1>
+            <button
+              id="copy"
+              phx-hook="CopyHook"
+              class="text-white bg-[#755FFF] py-1 px-2 rounded-lg text-[16px]  lg:text-[16px] md:text-[15px] sm:text-[14px]"
+            >
+              Copy
+            </button>
+          </div>
+          <h1 class="text-white lg:mt-4 md:mt-5 sm:mt-4">
+            2. Open Terminal by pressing <b class="font-extrabold">Ctrl + Alt + T</b> together
+          </h1>
+          <h1 class="text-white lg:mt-3 md:mt-5 sm:mt-4">
+            3. Paste the shell command by hitting <b>Ctrl + Alt + V</b> together
+          </h1>
+          <h1 class="text-white lg:mt-3 md:mt-5 sm:mt-4">
+            4. Run the command by hitting <b>ENTER</b>
+          </h1>
+        </div>
+      </div>
+      <.link navigate={~p"/"}>
```

Comment:
As all these should be only one LiveView, the attribute should be patch and not navigate.